executor/prepared_test.go

```diff
@@ -98,6 +98,19 @@ func TestPreparedStmtWithHint(t *testing.T) {
 	tk.MustQuery("execute stmt").Check(testkit.Rows("1"))
 }
 
+func TestPreparedCTEStmtWithHint(t *testing.T) {
```

Comment:
I think it can be appended into ```TestPreparedStmtWithHint```. We find that it takes too much time for the test to start. So we suggest adding the new cases to the old ones.

Reply:
haha, OK

Reply:
Addressed, PTAL